docs/GUIDE.md

```diff
@@ -0,0 +1,137 @@
+2. Open the project with Xcode.
+3. Import the Agora SDK plugin. The project includes the Agora SDK as a dependency package. As soon as Xcode loads the project, the package download should start automatically and finish in a few minutes.
+
+If the package did not load for some reason, e.g. networking issues, or you are setting up a brand new project of your own, see the following information to cover the package manager, CDN, and cocoapods.
+
+1. To manually add the package to your Xcode project, put the specific branch of the git location as the dependency:
+![](/docs/images/blog_avp1_branch.png "package location")
+2.  CDN download     [https://download.agora.io/null/Agora_Native_SDK_for_iOS_rel.v4.2.6.133_39484_FULL_20240126_0204_291749.zip](https://download.agora.io/null/Agora_Native_SDK_for_iOS_rel.v4.2.6.133_39484_FULL_20240126_0204_291749.zip)
+    
+3.  Cocoa Pods
+```Makefile  
+	pod 'AgoraRtcEngine_Special_iOS', '4.2.6.133.VISION'
+```
+## Running the App
+
+Fill in your test mode Agora app ID in the appID field in VisionVideoCallView.swift.
+
+![](/docs/images/blog_avp2_appid.png "appid entry")
+
+Build and run for either the Simulator or your physical VisionPro device. At the same time, prepare a second user to join the chat as a remote user.
+
+### Remote User Setup
+
+Use another device or use the Agora Web demo app for connecting to the video chat channel. The following screenshot shows a web user published video feed using the same App ID and channel name.
+
+![](/docs/images/blog_avp3_remoteuser.png "remote user")
+  
+
+### Running on Simulator
+
+Since there is no camera simulation in the simulator, your local video stream won’t be presented to the other users. However, you can still see other users who have enabled their cameras or custom video feeds on their devices. The following screenshot captures what you will see in the simulator.
+
+
+![](/docs/images/blog_avp4_sim.png "simulator")
+
+### Running on VisionPro Device
+
+Before building your project to the device, Add the following keys to your Info.plist file to request the necessary permissions:
+```xml
+<key>NSCameraUsageDescription</key>
+<string>camera for self capture</string>
+<key>NSMicrophoneUsageDescription</key>
+<string>mic for my voice</string>
+```
+  
+![](/docs/images/blog_avp5_plist.png "plist")
+
+You should confirm your permission to allow the app to use your VisionPro’s camera and microphone. Once you join the channel, you should see yourself in your avatar form and the remote user should show up on your screen!
+
+![](/docs/images/blog_avp6_device.gif "device")
+
+The Web user can also see you in your digital form and you can start chatting together now!
+
+![](/docs/images/blog_avp7_webuser.gif "web user")
+  
+
+## Basic Code Walkthrough
+
+Reading and understanding the code is never a boring task for developers! Here is a break-down of the project source so you can find the area of your interest quickly.
```

Comment:
This intro sounds a bit robotic, consider revising to better explain to the user that this next section explains the code.